Unable to compile linux template_release

### Tested versions

4.7 master

### System information

linux

### Issue description

An error occurred when compiling `template_release`:
```
platform/linuxbsd/godot_linuxbsd.cpp: In function 'int main(int, char**)':
platform/linuxbsd/godot_linuxbsd.cpp:75:17: error: 'printf' was not declared in this scope
   75 |                 printf("A CPU with SSE4.2 instruction set support is required.\n");
      |                 ^~~~~~
platform/linuxbsd/godot_linuxbsd.cpp:39:1: note: 'printf' is defined in header '<cstdio>'; this is probably fixable by adding '#include <cstdio>'
   38 | #include <clocale>
  +++ |+#include <cstdio>
   39 | #include <cstdlib>
scons: *** [bin/obj/platform/linuxbsd/godot_linuxbsd.linuxbsd.template_release.x86_64.o] Error 1
scons: building terminated because of errors.
```

### Steps to reproduce

For Linux compile `template_release`

### Minimal reproduction project (MRP)

N/A

--- a/platform/linuxbsd/godot_linuxbsd.cpp
+++ b/platform/linuxbsd/godot_linuxbsd.cpp
@@ -36,6 +36,7 @@
 #include <unistd.h>
 #include <climits>
 #include <clocale>
+#include <cstdio>
 #include <cstdlib>
 
 #if defined(ASAN_ENABLED)

